src/server/routes/homeController.ts

```diff
@@ -0,0 +1,125 @@
+import { Ticket } from '../models/Ticket';
+import { User } from '../models/User';
+import { Session } from '../models/Session';
+import { helpers } from '../services/helperFunctions';
+import { Op } from "sequelize";
+import { Request, Response} from 'express';
+
+
+  export const list =  async (req: Request, res: Response) => {
+      try {
+          let status: any;
+          if (req.query.status === undefined || req.query.status === 'active') {
+              status = helpers.status();
+          } else {
+              status.push(req.query.status);
+          }
```

Comment:
```suggestion
           const status = helpers.status(req.query);
```
move `if` into helper function